result/bgpv4prefix.yaml

```diff
@@ -2,14 +2,14 @@ components:
   schemas:
     Bgpv4Prefix.State.Request:
       description: >-
-        The request to retrieve IPv4 Prefix state of a BGPV4 Peer.
+        The request to retrieve IPV4 Prefix state of a BGPV4 Peer.
```

Comment:
Are we consistently using IPV4 or IPv4?  To me the latter seems more appropriate.  Don't care too much either ways, as long as we are consistent throughout the model.